setup.py

```diff
@@ -29,7 +29,7 @@
         'requests',
         'PyYAML',
         'cerberus',
-        'wget'
+        'progressbar'
```

Comment:
`progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

Reply:
done